run/convert.go

```diff
@@ -116,3 +125,85 @@ func filterPrimaryKeyColumns(columns []*data.Column) []*data.Column {
 
 	return pkColumns
 }
+
+func mapSchemaForeignKeyReferences(isch *database.Schema, sch *data.Schema) error {
+	for _, t := range isch.Tables {
+		table, ok := sch.TablesByName[t.Name]
+		if !ok {
+			log.Printf("Unmapped table %v in %v", t.Name, isch.Name)
+			continue
+		}
+
+		distinctForeignColumnsByForeignKeyName := map[string]*data.ForeignColumn{}
+
+		for _, c := range t.Columns {
+			column, ok := table.ColumnsByName[c.Name]
+			if !ok {
+				log.Printf("Unmapped column %v in %v.%v", c.Name, isch.Name, t.Name)
+				continue
+			}
+
+			if column.IsForeignKey {
+				foreignTable, ok := sch.TablesByName[c.ForeignKey.ForeignTableName]
+				if !ok {
+					log.Printf("Unmapped foreign table %v in %v", c.ForeignKey.ForeignTableName, isch.Name)
+					continue
+				}
+				foreignColumn, ok := foreignTable.ColumnsByName[c.ForeignKey.ForeignColumnName]
+				if !ok {
+					log.Printf("Unmapped foreign column %v in %v.%v", c.ForeignKey.ForeignColumnName, isch.Name, c.ForeignKey.ForeignTableName)
+					continue
+				}
+
+				fc := &data.ForeignColumn{
+					Name:                     c.ForeignKey.Name,
+					ColumnName:               column.Name,
+					ForeignColumnName:        column.Name,
```

Comment:
foreigncolumn.Name I think

Reply:
Note for myself: ForeignColumnName is grabbing the wrong value.